### PR TITLE
Fragmentation Effect/Munitions For The Third Fucking Time

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl: Surrealistik
add: Fragmentation explosives have their blast radius significantly reduced. In exchange they now discharge a blast wave of armour piercing flechette shrapnel in a semi-random scattershot around them.
add: Fragmentation rocket type added.
tweak: Base rocket direct hit damage increased to 50.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Conflicts and changelog documentation, please.

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -914,7 +914,7 @@
 	accuracy = config.max_hit_accuracy
 	accurate_range = config.norm_shell_range
 	max_range = config.long_shell_range
-	damage = config.med_hit_damage
+	damage = config.high_hit_damage
```
Comment (DominikPanic) [inline code/modules/projectiles/updated_projectiles/ammo_datums.dm]: Please document this change.

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -943,6 +943,26 @@
 	smoke.set_up(1, get_turf(P))
 	smoke.start()
 
+/datum/ammo/rocket/frag
+	name = "fragmentation rocket"
```
Comment (DominikPanic) [inline code/modules/projectiles/updated_projectiles/ammo_datums.dm]: Try to get a sprite for it for the ammo hud.

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm [hunk 3]
```diff
@@ -1602,3 +1622,22 @@
...
 	icon_state = "smoke_shell"
+
+
+/datum/ammo/bullet/shrapnel/flechette
+	name = "fragmentation flechette"
+	icon_state = "flechette"
+	flags_ammo_behavior = AMMO_SHRAPNEL
+
+/datum/ammo/bullet/shrapnel/flechette/New()
+	..()
```
Comment (DominikPanic) [inline code/modules/projectiles/updated_projectiles/ammo_datums.dm]:
```suggestion
	. = ..()
```

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm (hunk 2 above)
Comment (Surrealaser) [reply code/modules/projectiles/updated_projectiles/ammo_datums.dm]: done

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm (hunk 1 above)
Comment (Surrealaser) [reply code/modules/projectiles/updated_projectiles/ammo_datums.dm]: done

**Comment by DominikPanic** [comment]
Conflicts. This will also need test-merging as we were unable to test it due to TM conflicts.

**Comment by spookydonut** [comment]
#73

**Pull request closed without merging**